docs/references/studio/api/datasets.md

```diff
@@ -0,0 +1,257 @@
+
+### List Datasets
+
+**Description**: Retrieve a list of datasets with optional prefix filtering.
+
+**Endpoint**: `/datachain/datasets`
+
+**Method**: `GET`
+
+**Authentication**: Requires token with `read` access to `DATASETS` scope from [`datachain auth login`](../../../commands/auth/login.md)
+
+**Parameters**:
+
+- `prefix` (query, optional): Filter datasets by name prefix
+- `team_name` (query, required): Team identifier (automatically added by client)
+
+**Example Request**:
+```bash
+curl -X GET "https://studio.datachain.ai/api/datachain/datasets?prefix=training&team_name=my-team" \
+  -H "Authorization: token YOUR_STUDIO_TOKEN" \
+  -H "Content-Type: application/json"
+```
+
+**Example Response**:
+```json
+[
+  {
+    "attrs": ["training"],
+    "created_at": "2025-06-11T14:22:31.123456+00:00",
+    "created_by_id": 2,
+    "description": "Training dataset for computer vision",
+    "id": 1853,
+    "name": "training-images-v2",
+    "project": {
+      "created_at": "2025-06-20T04:34:24.083394+00:00",
+      "created_by_id": null,
+      "descr": "Computer Vision Project",
+      "id": 2,
+      "name": "cv-project",
+      "namespace": {
+        "created_at": "2025-06-20T04:34:24.074155+00:00",
+        "created_by_id": null,
+        "descr": "CV Team Namespace",
+        "id": 2,
+        "name": "cv-team",
+        "team_id": 1,
+        "uuid": "b73d73f9-420a-5314-0994-7859bd067c19"
+      },
+      "uuid": "ddd29091-4578-55f5-c1d2-77651d997f2"
+    },
+    "team_id": 1,
+    "versions": [
+      {
+        "created_at": "2025-06-11T14:22:31.234567+00:00",
+        "created_by_id": 2,
+        "dataset_id": 1853,
+        "error_message": "",
+        "error_stack": "",
+        "finished_at": "2025-06-11T14:23:45.345678+00:00",
+        "id": 4150,
+        "job_id": null,
+        "num_objects": 1000,
+        "query_script": "",
+        "size": 5000000,
+        "status": 4,
```

Comment:
Do we have these `status` numbers described somewhere? May be it is better to return strings (status labels) instead of IDs?